examples/experimental/dagster-airlift/examples/tutorial-example/tutorial_example/dagster_defs/stages/standalone.py

```diff
@@ -59,20 +59,19 @@ def rebuild_customers_list_defs() -> Definitions:
                 table_name="customers",
                 csv_path=Path(os.environ["TUTORIAL_EXAMPLE_DIR"]) / "customers.csv",
                 duckdb_path=Path(os.environ["AIRFLOW_HOME"]) / "jaffle_shop.duckdb",
-                duckdb_schema="raw_data",
                 duckdb_database_name="jaffle_shop",
             ),
         ),
     )
 
     rebuild_customers_list_schedule = ScheduleDefinition(
         name="rebuild_customers_list_schedule",
-        target=AssetSelection.assets(*asset_defs.assets),  # type: ignore
+        target=AssetSelection.assets(merged_defs.get_asset_graph().all_asset_keys),  # type: ignore
```

Comment:
whats up with the type ignore?

Reply:
dropped it - this does have to be list-ified though if you check the newest push